python/tank/descriptor/io_descriptor/git_tag.py

```diff
@@ -215,13 +208,14 @@ def _fetch_tags(self):
             # clone the repo, list all tags
             # for the repository, across all branches
             commands = ["ls-remote -q --tags %s" % self._path]
-            tags = self._tmp_clone_then_execute_git_commands(commands, depth=1).split(
-                "\n"
-            )
+            output = self._tmp_clone_then_execute_git_commands(commands, depth=1)
+            if isinstance(output, bytes):
+                output = output.decode("utf-8")
```

Comment:
I'm not sure about utf-8 here. On Windows, the default encoding might different. Like [CP-1252](https://en.wikipedia.org/wiki/Windows-1252)